docs/source/quickstart.rst

```diff
@@ -53,14 +53,14 @@ Then we can use the NSAlert object::
 
     from pyobjus import autoclass
 
+    # mimic the constant string operator (@"hello") in objective C
+    from pyobjus import objc_str
```

Comment:
https://github.com/kivy/pyobjus/blob/6a2c320b531982ac382183318415d2aa68982d46/pyobjus/pyobjus_types.pxi#L350

this is not the original thing which `ns` variable calls. Instead of rewriting the original one just add the code and provide the difference between the old and new behavior instead of just rewriting the old docs.